packages/editor/src/lib/editor/Editor.ts

```diff
@@ -5630,11 +5649,11 @@ export class Editor extends EventEmitter<TLEventMap> {
 				.filter((shape) => {
 					if (!shape) return false
 
-					if (this.isShapeOfType<TLArrowShape>(shape, 'arrow')) {
-						if (shape.props.start.type === 'binding' || shape.props.end.type === 'binding') {
-							return false
-						}
-					}
+					// if (this.isShapeOfType<TLArrowShape>(shape, 'arrow')) {
+					// 	if (shape.props.start.type === 'binding' || shape.props.end.type === 'binding') {
+					// 		return false
+					// 	}
+					// }
```

Comment:
hm we probably should still be ignoring arrows if they are bound to other shapes. Would a new util flag make sense here? `canBeLaidOut`